README.md

```diff
@@ -32,6 +32,7 @@ When adding diffs into markdown, use the following format:
 
 - Specify the code block is a diff with `diff` on the same line as the backticks.
 - Specify the base git reference the diff applies to, either a tag or a commit sha, using the `mddiffcheck.base=` parameter.
+- If you want to ignore a specific diff code block you can use the `mddiffcheck.ignore=true` parameter.
```

Comment:
I intentionally didn't mention this feature because I really didn't want to add it and only added it because of diffs in CAPs I couldn't resolve. But you're right, if it is a feature, it should be documented, and tested for that matter.
```suggestion
- If necessary, specify the diff should be ignored and not checked, using the `mddiffcheck.ignore=true` parameter.
```

Reply:
https://github.com/stellar/mddiffcheck/pull/3/commits/4853d7c9816ef8f4b292404f536700e1c40d6f94